package.json

```
@@ -1,9 +1,11 @@
 {
   "name": "stockx-api",
-  "version": "1.0.3",
-  "description": "communicate with the unofficial StockX API",
-  "main": "index.js",
+  "version": "1.0.0",
```

```suggestion
  "version": "1.0.4",
```
Probably will have to bump this to `1.0.4` since it looks like the most recent version (located on npm) was 1.0.3

---

If you want, we can also do a minor version change (1.1.0)